### PR TITLE
add tuneR, seewave, soundecology extensions for R 3.4.0 (+ libsndfile as dep)

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86 please review?

File: easybuild/easyconfigs/r/R/R-3.4.0-intel-2017a-X11-20170314.eb [hunk 1]
```diff
@@ -44,6 +44,8 @@ dependencies = [
     ('PROJ', '4.9.3'),  # for rgdal
     ('GMP', '6.1.2'),  # for igraph
     ('NLopt', '2.4.2'),  # for nloptr
+    ('FFTW', '3.3.6'),  # for fftw
```
Comment (wpoely86) [inline easybuild/easyconfigs/r/R/R-3.4.0-intel-2017a-X11-20170314.eb]: why not the MKL one?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2069.delcatty.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/6d56bae44e25b971395463f6abf9f105 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2427.golett.os - Linux centos linux 7.3.1611, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/2599dca9453a42cd77166bdeccd9912f for a full test report.

File: easybuild/easyconfigs/r/R/R-3.4.0-intel-2017a-X11-20170314.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/r/R/R-3.4.0-intel-2017a-X11-20170314.eb]: I don't think the `fftw` R wrapper supports it

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**